packages/fxa-payments-server/package.json

```diff
@@ -105,6 +105,7 @@
     "prettier": "^2.3.1",
     "redux-devtools-extension": "^2.13.9",
     "sinon": "^9.0.3",
+    "storybook-addon-mock": "2.4.1",
```

Comment:
We need to use v2.4.1 for now, since we still have some pre-Storybook v6 storybooks.